src/java.desktop/share/classes/sun/swing/SwingAccessor.java

```diff
@@ -47,6 +56,26 @@ public final class SwingAccessor {
     private SwingAccessor() {
     }
 
+    public static AccessibleComponentAccessor getAccessibleComponentAccessor() {
+        return accessibleComponentAccessor;
```

Comment:
It is better to force the "ensureClassInitialized" as any other getters in this class. To make sure the field is up-to-date even if it was initialized on another thread.

Reply:
Done